product-scenarios/scenarios-commons/src/main/java/org/wso2/carbon/esb/scenario/test/common/utils/XMLComparatorInterop.java

```diff
@@ -0,0 +1,212 @@
+package org.wso2.carbon.esb.scenario.test.common.utils;
+
+import org.apache.axiom.om.OMAttribute;
+import org.apache.axiom.om.OMElement;
+import org.apache.axiom.om.OMNamespace;
+import org.apache.axiom.om.OMNode;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import java.util.Iterator;
+import java.util.Vector;
+
+/**
+ * Util class for comparing two OMElements
+ */
+public class XMLComparatorInterop {
+
+    private static final Log log = LogFactory.getLog(XMLComparatorInterop.class);
+    //if namespaces needs to be ignored need to add to this list
+    private static Vector ignorableNamespaceList = new Vector();
+
+    /**
+     * Compares two OMElements. It compares attributes, namespaces, elements with trimmed values.
+     *
+     * @param elementOne first element to compare
+     * @param elementTwo element to compare with the first
+     * @return true if compares
+     */
+    public static boolean compare(OMElement elementOne, OMElement elementTwo) {
+
+        boolean status;
+        //ignore if the elements belong to any of the ignorable namespaces list
+
+        if (isIgnorable(elementOne) ||
+                isIgnorable(elementTwo)) {
```

Comment:
This line can be merged with the previous line

Reply:
Done.